drivers/spi/spi_ll_stm32.c

```diff
@@ -807,6 +807,38 @@ static bool spi_buf_set_in_nocache(const struct spi_buf_set *bufs)
 }
 #endif /* CONFIG_SOC_SERIES_STM32H7X */
 
+#ifdef CONFIG_SPI_STM32F7_ERRATA_BUSY
```

Comment:
Can this be put inside the function to factorize some lines ?

Reply:
done